components/brave_ads/common/pref_names.h

```diff
@@ -23,6 +23,10 @@ extern const char kOnboardingTimestamp[];
 
 extern const char kShouldShowMyFirstAdNotification[];
 
+extern const char kSupportedRegionsLastSchemaVersion[];
+extern const char kSupportedRegionsSchemaVersion[];
+extern const int kSupportedRegionsSchemaVersionNumber;
+
 extern const char kVersion[];
 extern const int kCurrentVersionNumber;
```

Comment:
the same here, I would do unsigned int

Reply:
@SergeyZhukovsky `PrefRegistry` does not have an unsigned version of `RegisterIntegerPref`, see https://chromium.googlesource.com/chromium/src.git/+/72.0.3626.80/components/prefs/pref_registry_simple.h?autodive=0%2F%2F%2F